pkgs/applications/editors/neovim/build-neovim-plugin.nix

```diff
@@ -26,7 +26,7 @@ in
           lua_modules_path = "lua"
         '';
       });
-      finalDrv = toVimPlugin (luaDrv.overrideAttrs(oa: {
+      finalDrv = toVimPlugin (luaDrv.overrideAttrs(oa: (builtins.removeAttrs attrs ["version"]) // {
```

Comment:
removing the version number is also confusing because the derivation name will use a version different than the src's but keeping it breaks luarocks builder. I need to address that.